blueprints/hr_assistant/domains/date/get_date/train.txt

```diff
@@ -0,0 +1,203 @@
+Give me {mia|name}'s {hiring date|employment_action}
+{Catherine Ybarra|name}'s {bday|dob} is in which month
+Please Get me {Kelley Spirea|name}'s {Bday|dob}
+Was {mia brown|name} {fired|employment_action} in between the month of June and August?
+tell me {birth date|dob} of {nan|name}
+What is the duration of {Nan|name}'s employment with the company?
+{Theresa Tinto|name} {birthday|dob}
+When was {ivan rogers|name} {born|dob}?
+When was {Carla Demita|name}'s {join|employment_action} date?
+{Michael Albert|name} {birthday|dob}
+What day was it when {Amy|name} was {hired|employment_action}
+How long has it been since {Lenora Tejeda|name} {started working here|employment_action}?
+The date was what when {Ivan Rogers|name} {started working|employment_action} for the company
+What year did {Hang Wolk|name} {end his employment|employment_action}?
+When did {Ricardo Ruiz|name}'s {start working|employment_action}?
+What is the {date of hiring|employment_action} for {kamrin|name}
+How long was {Sneha|name} working for?
+When {ivan rogers|name} {began her employment|employment_action}, what was the date
+how long is it that {george|name} has {been with the company|employment_action}?
+{Amy|name} {hiring date|employment_action}
+When {Sophia Theamstern|name} was {hired|employment_action} to work at the company, what was the date?
+{Rick Clayton|name} {worked here|employment_action} for how long
+When did {Mia Brown|name} end her employment
+Give me the {bday month|dob} of {Christie Harrington|name}
+When did {Seffi Shields|name} get {fired|employment_action} from the company?
+Please get me {DOB|dob} of {Ludwick Harrell|name}
+When was {Joe South|name} {date of starting employment|employment_action}?
+Was {Ivan|name} {fired on|employment_action} the same month that she was {hired|employment_action}?
+The date was what when {Allison Lydon|name} began her {experience|employment_action} at the company
+Has {Kristen Squatrito|name} been working here for {6 years|sys_duration}?
+{fire date|employment_action}, {Ivan|name}
+{Violeta Ferreira|name} {started|employment_action} when
+{mia brown|name}'s {bday|dob} is in which month
+How long has {Jyoti Lajiri|name} been working here?
+What was the date {Brandon LeBlanc|name} {start date|employment_action}
+{date of birth|dob} of {Wilson Adinolfi|name}
+Please get me {DOB|dob} of {Julissa Hunts|name}
+How long has {Simon Roup|name} {worked here|employment_action} and was it the same month that she was {hired|employment_action}?
+Give me the {bday month|dob} of {Ludwick Harrell|name}
+What is {Bonalyn|name}s number of years {of experience|employment_action} at this company?
+Give me the {bday month|dob} of {Ivan Rogers|name}
+When did {Eric|name} {start|employment_action} working at the company?
+in what month was {Maryellen Jackson|name} {born|dob}
+When was {Caitrin Strong|name} {date of hire|employment_action}?
+How long has {Nina Panjwani|name} been with the company?
```

Comment:
{been with the company|employment_action}?
Need to look into this file to make entity annotations more consistent.